schemas/isd.json

```diff
@@ -0,0 +1,242 @@
+	      "coefficients": {
+	         "type": "array",
+		 "items": {
+		    "type": "number"
+		  }
+		},
+	        "key" : {
+		    "type": "string",
+		    "description": "key into LUT to determine equation to use"
+		    }
+	 },
+	"required": ["coefficients", "equation_key"]
+       },
+
+       "STARTING_DETECTOR_SAMPLE": {
+           "description": "The sample on the detector where the data starts",
+           "type": "number"
+       },
+       "STARTING_DETECTOR_LINE": {
+           "description": "The line on the detector where the data starts",
+           "type": "number"
+       },
+       "FOCAL2PIXEL_X": {
+           "description": "Transformation x coefficients from focal plane (mm) to pixel coordinates",
+           "type": "array",
+	          "items": {
+                 "type": "number"
+            }
+       },
+       "FOCAL2PIXEL_Y": {
+           "description": "Transformation y coefficients from focal plane (mm) to pixel coordinates",
+           "type": "array",
+	          "items": {
+                 "type": "number"
+            }
+       },
+       "SENSOR_LOCATION": {
+           "description": "(X, Y, Z) location of sensor in body-fixed frame.",
+           "type": "array",
+	          "items": {
+               "type": "array",
+	              "items" : {
+                  "type": "number",
+                  "minItems": 3,
+                  "maxItems": 3
+		              }
+	               }
+      },
+       "SENSOR_VELOCITY" : {
+           "description": "(v_x, v_y, v_z) velocity of sensor in body-fixed frame.",
+           "type": "array",
+	          "items": {
+               "type": "array",
+	              "items" : {
+                    "type": "number",
+                    "minItems": 3,
+                    "maxItems": 3
+		                }
+	                 }
+	     },
+       "SUN_POSITION" : {
+           "description": "(v_x, v_y, v_z) velocity of sun in body-fixed frame.",
+           "type": "array",
+	          "items": {
+               "type": "array",
+	              "items" : {
+                  "type": "number",
+                  "minItems": 3,
+                  "maxItems": 3
+		              }
+	               }
+	     },
+       "SUN_VELOCITY" : {
+           "description": "(v_x, v_y, v_z) velocity of sun in body-fixed frame.",
+           "type": "array",
+	          "items": {
+               "type": "array",
+	              "items" : {
+                  "type": "number",
+                  "minItems": 3,
+                  "maxItems": 3
+		        }
+	         }
+	     },
+
+       "ORIENTATION" : {
+           "description": "Orientation of the sensor provided in quaternions",
+           "type": "array",
+           "items": {
+               "type": "array",
+               "items" : {
+                  "type": "number",
+                  "minItems": 4 ,
+                  "maxItems": 4
+           }
+          }
+      },
+
+       "DETECTOR_LINE_RATE": {
```

Comment:
I'm not sure if this is the correct name for this. Previously this was Integration Time. ISIS calls this Line Rate. It's also been called Exposure Rate and Line Scan Rate. I lean towards Line Scan Rate, but I'm not sure how well that matches image labels.

Reply:
I'm not sure what the best name for this is either. `LINE_SCAN_RATE` sounds good to me.

Reply:
It's also worth noting that exposure time and integration time are generally the same thing, but scan rate could be different.  For some line scan cameras, there's a small but relevant window of time in-between the end of one line and the start of the next.

Reply:
The values in this table are kind of used as both.

Reply:
LINE_SCAN_RATE works for me. In all descriptions for "RATE"s can we list the unit of time (e.g. in seconds).

Reply:
Yes, that's a good point. We'll add 'seconds' to the description.